weedcoco/index/indexing.py

```diff
@@ -18,7 +18,6 @@ class ElasticSearchIndexer:
         weedcoco_path (pathlib.PosixPath): weedcoco local path for indexing
         thumbnail_dir (pathlib.PosixPath): thumbnail folder local path to be integrated into the json file for indexing
         es_index_name (string): index name for ElasticSearch
-        es_type_name (string): type name for ElasticSearch
```

Comment:
are you sure it's safe to remove this on prod?

Reply:
The type has been deprecated, but I think I removed it because last time I thought it was the issue of bug (the geo type location cannot be indexed and it was giving me the warning of type being deprecated). We can either keep it or remove it. They work exactly the same in our case. The current index on production doesn't have this type.